lib/signaling/v2/iceconnectionmonitor.js

```diff
@@ -46,13 +46,7 @@ class IceConnectionMonitor {
   _getIceConnectionStats() {
     return this._peerConnection.getStats().then(stats => Array.from(stats.values()).find(stat => {
       return stat.type === 'candidate-pair' && stat.nominated;
-    })).then((activeCandidatePairStats) => {
-      return {
-        timestamp: activeCandidatePairStats.timestamp,
-        bytesReceived: activeCandidatePairStats.bytesReceived,
-        bytesSent: activeCandidatePairStats.bytesSent
-      };
-    }).catch(() => {
+    })).catch(() => {
```

Comment:
@makarandp0 I'll leave it up to you, but the catch callback can also be the second argument for `then()`:
```js
.then(onResolved, onRejected);
```

Reply:
@manjeshbhargav, I do see usage of 2nd param of `.then` in our codebase, and thinking we should move to using .catch instead - 1st type of usage could be problematic. More about this antipattern at https://www.datchley.name/promise-patterns-anti-patterns/